packages/rancher-logging/overlay/templates/loggings/rke1/logging-rke1.yaml

```diff
@@ -4,7 +4,8 @@ kind: Logging
 metadata:
   name: {{ .Release.Name }}-rke
   namespace: {{ .Release.Namespace }}
-  labels: {{ include "logging-operator.labels" . | indent 4 }}
+  labels:
+{{ include "logging-operator.labels" . | indent 4 }}
```

Comment:
separate lines necessary?

Reply:
Yes I get template errors if it is on the sameline